docs/source/about/guides/windows/discord_voicemeeter.rst

```diff
@@ -0,0 +1,35 @@
+Discord call audio cancellation with Voicemeeter (Standard)
+===========================================================
+
+Voicemeeter
+^^^^^^^^^^^
+
+#. Click Hardware Out
+#. Set the physical device you recieve audio to as your Hardware Out with MME
+#. Turn on BUS A for the Virtual Input
+
+Windows
+^^^^^^^
+
+#. Open the sound settings
+#. Set your default Playback as Voicemeeter Input
+
+.. note:: Run audio in the background to find the device that your Virtual Input is using (Voicemeeter In #), you will see the bar to the right of the device have green bars going up and down. This device will be referred to as Voicemeeter Input.
```

Comment:
```suggestion
.. note:: Run audio in the background to find the device that your Virtual Input is using
   (Voicemeeter In #), you will see the bar to the right of the device have green bars
   going up and down. This device will be referred to as Voicemeeter Input.
```

Let's keep line length under 120 characters.

Reply:
Sorry about that, I was attempting to figure that out since I am new to this.

Would this be better?
```suggestion
.. note::
    Run audio in the background to find the device that your Virtual Input is using. (Voicemeeter In #)
    You will see the bar to the right of the device have green bars going up and down.
    This device will be referred to as Voicemeeter Input.
```

Reply:
Sorry if my suggestion wasn't the clearest. You don't have to break each line on the end of  a sentence or anything like that. The periods don't matter. My suggestion will still render it the same way as if it were all on one line, or one paragraph. It just makes it easier to read in an IDE.

Reply:
I see what you mean, thanks for the clarification. I will commit your suggestion.